### ISSUE TITLE
prereq checks for cdsw

### ISSUE THREAD

**Issue by manojsundaram** [cloudera-ps/prereq-checks#90]
Add checks for Cloudera data science workbench, prereqs

**Comment by melvin-koh** [comment]
+1

Suggested checks:
1. Check localhost resolve to 127.0.0.1
2. Check wilcard dns is working
3. Check that block device for docker images is not partitioned, formatted, mounted etc.
4. Check that /var/lib/cdsw is on a dedicated disk

**Comment by melvin-koh** [comment]
Output of the CDSW check:

```
Cloudera Prerequisites Checks v1.5

System information
------------------
FQDN:          hdredirect-lb6-54290b28133ca5af.elb.us-east-1.amazonaws.com
Distro:        CentOS 7.3
Kernel:        3.10.0-514.el7.x86_64
CPUs:          1x Intel i7-4770HQ @ 2.20GHz
RAM:           0.95G
Timezone:      SGT
DateTime:      Thu Jan 31 11:29:10 SGT 2019
nsswitch:      files dns myhostname
DNS server:    192.168.246.12
Block Devices:
               /dev/sdc	  5G	GPT
               /dev/sdd	  5G	MBR
               /dev/sde	  5G	None

CDSW prerequisite checks
------------------------
 PASS  System: user id 8536 is not in use
 PASS  Network: localhost correctly resolves to 127.0.0.1
 PASS  Network: wildcard DNS correctly resolves to CDSW Master IP 192.168.246.180
 FAIL  System: Application block device for /var/lib/cdsw not found
```

**Comment by melvin-koh** [comment]
- Added iptables check
- Tested on CentOS 7.5, SLES 12
